Source/Scene/Model.js

```diff
@@ -2931,7 +3008,28 @@ define([
 
                 var castShadows = ShadowMode.castShadows(model._shadows);
                 var receiveShadows = ShadowMode.receiveShadows(model._shadows);
-                
+
+                // Setup the stencil for the first pass
+                var drawRS = clone(rs);
+                drawRS.stencilTest = {
```

Comment:
Here and below remove the [defaults](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Renderer/RenderState.js#L151-L166) unless it makes it too hard to follow.